index.js

```diff
@@ -32,9 +42,20 @@ getFullNames(usersArray);
 
 const getUsersCreditDetails = arr => {
   // Your code goes here ...
+  const usersCreditDetails = [];
+  for(let {firstName, lastName, balance} of arr){
+    const userDetails = {
+      firstName,
+      lastName,
+      balance
+    }
+    usersCreditDetails.push(userDetails)
```

Comment:
here because you have already destructuered the values in the theres no need to "repackage" them into a new object just to push them.

For readability and usability, what I would recommend is to do the for loop with a whole object:
```js
for (let user of arr) {
```
and then destructure and just send those values:
```js
const { firstName, lastName, balance} = user;
usersCreditDetails.push({ firstName, lastName, balance});
```

Reply:
I see what you mean with this however, the expected output is the following..

Reply:
I see that, once again, it works 👍 
the way I am pushing is, itself, a new object